ui/app/components/clients/dashboard.js

```diff
@@ -35,6 +35,7 @@ export default class Dashboard extends Component {
 
   @service store;
 
+  @tracked queriedActivityResponse = null;
```

Comment:
maybe put these in alphabetical order? I think we started that way but then it got all messed up.

Reply:
YES - I was waiting to alphabetize because I wanted to clean-up and remove tracked properties we were no longer using (to avoid changing the order and creating unnecessary git conflicts). I'll add a comment TODO to make sure it gets done